spec/features/consumer/shopping/embedded_groups_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+feature "Using embedded shopfront functionality", js: true do
+
+  Capybara.server_port = 9999
+
+  describe 'embedded groups' do
+	let(:enterprise) { create(:distributor_enterprise) }
+  	let!(:group) { create(:enterprise_group, enterprises: [enterprise], permalink: 'group1', on_front_page: true) }
+
+
+  	before do
+  	  Spree::Config[:enable_embedded_shopfronts] = true
+      Spree::Config[:embedded_shopfronts_whitelist] = 'localhost'
```

Comment:
Just a minor thing: the whitespaces don't seem to be right. Tabs and spaces are mixed. Our convention is to use spaces only (no tabs) to preserve the same indentation no matter which tool you use.